### PR TITLE
3222 coverage follow up

### PR DESCRIPTION
**Pull request by flooie**
PR to address issue #3222 

Updates for 

1. Performance enhancements using SOLR
2. Mobile - fixes for TOC and charts (hidden)
3. Architecture - dropped sending court names to API 
4. Content: has been added

Did not add spinner (hoping speed resolves that) 
Same for caching.  I think this may be fast enough.

### CODE REVIEW

File: cl/api/views.py
Comment (mlissner) [inline cl/api/views.py:1]: This looks pretty OK to me, but could we move the `if query_parameters` outside the session context handler and change it to an `if not` statement that early aborts the function? That'd help with indentation.

I'd also like one of our search gurus (we have those now!) to take a look. @albertisfu, does the search part of this PR look OK to you?

File: cl/assets/static-global/css/override.css [hunk 1]
```diff
@@ -1564,3 +1564,11 @@ textarea {
...
   height: 100vh;
 }
+
+@media (max-width: 780px) {
+    #toc-container {
+        overflow-y: initial;
+        position: relative;
+        height: min-content;
+    }
+}
```
Comment (mlissner) [inline cl/assets/static-global/css/override.css:1574]: Can you explain what this does and maybe post some screenshots, please?

**Review by mlissner — COMMENTED** [review]
Looks pretty good. Why not add caching though? We know these are still pretty mean queries. Caches would make life a lot faster and are easy to add?

File: cl/assets/static-global/css/override.css (hunk 1 above)
Comment (flooie) [reply cl/assets/static-global/css/override.css:1574]: This handles the table of contents displaying on top of the other content.  

On mobile we want the table of contents to just work and not be a scrollable list- so we need to stretch it out.

Ill add some photos

Comment (flooie) [reply cl/assets/static-global/css/override.css:1574]: Screenshot 2023-10-04 at 3 49 14 PM

This is attempting to resolve this on mobile 


Screenshot 2023-10-04 at 3 49 29 PM


This removes the sticky sidebar - and the overlapping text issue and makes it much more usable.

File: cl/api/views.py
Comment (flooie) [reply cl/api/views.py:1]: @albertisfu - thanks for taking a look at this.  I look forward to your thoughts.  I can move the if statement around but we really shouldn't be seeing this very often where we need to short circuit the statement

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
@ERosendo, just a heads up, we added a new function here that hits Solr. I'm guessing you'll want to convert it to an Elastic query.

File: cl/api/views.py
Comment (albertisfu) [reply cl/api/views.py:1]: Yeah, sorry for not having looked at this earlier. I just reviewed it in a call with Mike, and the query seems pretty right to me.

**Pull request merged**